File: src/main/kotlin/com/example/elasticsearch/repositories/LogDataRepository.java
package com.example.elasticsearch.repositories;

import com.example.elasticsearch.model.LogData;
import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;

import java.util.List;

public interface LogDataRepository extends ElasticsearchRepository<LogData, String>, CustomLogDataRepository {
    List<LogData> findByDescription(String description);
}

File: src/main/kotlin/com/example/elasticsearch/repositories/LogDataRepository.kt
package com.example.elasticsearch.repositories

import com.example.elasticsearch.model.LogData
import org.springframework.data.elasticsearch.repository.ElasticsearchRepository

interface LogDataRepository : ElasticsearchRepository<LogData?, String?>, CustomLogDataRepository {
    fun findByDescription(description: String?): List<LogData?>
}